src/allowed_args.cpp

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) 2017 The Bitcoin Classic developers
+// Distributed under the MIT software license, see the accompanying
```

Comment:
You can use MIT just fine, no worries. Its a personal preference and I wanted to make sure you knew you had a choice.

The licenses are compatible, as they should be because when we link to the static build of Qt we link bitcoin classic to the LGPLv3 licensed Qt libraries. Static linking means its to be interpreted as the GPLv3.
The FSF website also has a page on licenses compatible with the GPL.